docs/pages/inboxes/manage-inboxes.mdx

```diff
@@ -208,3 +208,37 @@ The wallet address used to initiate a client should be matched to its original i
 You do have the ability to rotate inbox IDs if a user reaches the limit of 257 identity actions (adding, removing, or revoking addresses or installations). Hopefully, users won’t reach this limit, but if they do, inbox IDs have a nonce and can be created an infinite number of times. 
 
 However, anytime a new inbox ID is created for an address, the conversations and messages in any existing inbox ID associated with the address are lost.
+
+### I have multiple addresses associated with one inbox ID. If I log in with any one of these addresses, does it access that inbox ID, or does it create a new inbox ID?
+
+Let's say that you create a client with an address 0x62EE...309c. Inbox ID 1 is generated from that address 0x62EE...309c. 
+
+<div>
+<img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/refs/heads/main/docs/pages/img/inbox1-address1.png" width="150px" />
+</div>
+
+If you then add a different address 0xd0e4...DCe8 to inbox ID 1, address 0xd0e4...DCe8 is also associated with that inbox ID 1, even inbox ID 1 can not be generated from address 0xd0e4...DCe8.
```

Comment:
```suggestion
If you then add a different address 0xd0e4...DCe8 to inbox ID 1, address 0xd0e4...DCe8 is also associated with that inbox ID 1.
```

I think we can remove this last clause. Inbox ID 1 is already generated at this step, so I dont think it needs to be stated whether it can be generated by the new address.